sal_interfaces/MTM2/MTM2_Telemetry.xml

```diff
@@ -272,6 +327,54 @@
     </item>
 </SALTelemetry>
 
+<SALTelemetry>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_balanceForceData</EFDB_Topic>
+    <Description>Net forces and moments as commanded by the force balance system. This is a component of the total force, which has the topics of netForcesTotal and netMomentsTotal.</Description>
+    <item>
+        <EFDB_Name>fx</EFDB_Name>
+        <Description>Net force in x direction.</Description>
```

Comment:
suggest `total x-force commanded by the force balance system`. Same for the 5 other parameters below.

Reply:
Done in commit: 9c5cc3f. Thanks!